docs/dev/setup.sh

```diff
@@ -1,34 +1,56 @@
 #!/bin/sh
 
-stream_backend="$1"
-
-if [ -z "$stream_backend" ] || (echo "stan jetstream" | grep -v -q "$stream_backend"); then
-  stream_backend="stan"
-fi
-echo "Using $stream_backend"
-
+OSTYPE=$(uname -s)
 COMPOSE_CMD='docker-compose'
+CLUSTER_NAME="demo-a"
+COMPOSE_FILE="./docs/dev/armada-compose.yaml"
 
-DCLIENT_VERSION=$(docker version -f '{{.Client.Version}}')
-if echo $DCLIENT_VERSION | grep -q '^2' ; then
+docker compose > /dev/null 2>&1
+if [ $? -eq 0 ] ; then
```

Comment:
nit: you can delete line 8 and instead: 

```suggestion
if docker compose >/dev/null 2>&1 ; then
```

but then you'd have to reverse the logic so... probably not worth it?

Reply:
Hm, yeah - I think reversing the logic could potentially be a bit confusing for devs taking a quick first glance at it.